### PR TITLE
misc: remove all appveyor references

### PR DESCRIPTION
**Pull request by patrickhulce**
**Summary**
Standing on the shoulders of giants (https://github.com/GoogleChrome/lighthouse/pull/11087 ,https://github.com/GoogleChrome/lighthouse/pull/10620, https://github.com/GoogleChrome/lighthouse/pull/11009), this PR removes our AppVeyor configuration to eliminate the last red check mark in our status checks :D 🎉 

RIP AppVeyor 💀 

https://github.com/GoogleChrome/lighthouse/pull/11087 well covers us at this point for Windows and since the rate-limiting fiasco was resolved it has been very fast with minimal queuing.

### CODE REVIEW

**Comment by connorjclark** [comment]
cmd+f AppVeyor

build status badge, some env vars.

**Comment by patrickhulce** [comment]
if we're ready to completely purge? :D aye aye 🖖

**Comment by patrickhulce** [comment]
oh it's so beautiful 😍 
image

File: lighthouse-core/scripts/download-chrome.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 # Download chrome inside of our CI env.
 
-if [ "$APPVEYOR" == "True" ]; then
```
Comment (brendankenny) [inline lighthouse-core/scripts/download-chrome.sh:11]: lol that wasn't a great check

File: readme.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Lighthouse  [![Linux Build Status](https://img.shields.io/travis/GoogleChrome/lighthouse/master.svg)](https://travis-ci.org/GoogleChrome/lighthouse) [![Windows Build Status](https://ci.appveyor.com/api/projects/status/46a5yoqc3hk59bl5/branch/master?svg=true)](https://ci.appveyor.com/project/paulirish/lighthouse/branch/master) [![Coverage Status](https://img.shields.io/coveralls/GoogleChrome/lighthouse/master.svg)](https://coveralls.io/github/GoogleChrome/lighthouse?branch=master) [![NPM lighthouse package](https://img.shields.io/npm/v/lighthouse.svg)](https://npmjs.org/package/lighthouse)
+# Lighthouse  [![GitHub Actions Status Badge](https://github.com/GoogleChrome/lighthouse/workflows/💡🏠/badge.svg)](https://github.com/GoogleChrome/lighthouse/actions) [![Linux Build Status](https://img.shields.io/travis/GoogleChrome/lighthouse/master.svg)](https://travis-ci.org/GoogleChrome/lighthouse) [![Coverage Status](https://img.shields.io/coveralls/GoogleChrome/lighthouse/master.svg)](https://coveralls.io/github/GoogleChrome/lighthouse?branch=master) [![NPM lighthouse package](https://img.shields.io/npm/v/lighthouse.svg)](https://npmjs.org/package/lighthouse)
```
Comment (brendankenny) [inline readme.md:1]: > `(https://github.com/GoogleChrome/lighthouse/workflows/💡🏠/badge.svg)](https://github.com/GoogleChrome/lighthouse/actions)`

Yay, but this is where we start to regret our workflow name. They still haven't fixed handling of emoji in their badge generator 😢 

bad badge screenshot

I guess we can pretend that little bit of the lightbulb peeking out isn't there and it's just a nameless badge until they fix things :)

Comment (patrickhulce) [reply readme.md:1]: > I guess we can pretend that little bit of the lightbulb peeking out isn't there and it's just a nameless badge until they fix things

LOL I didn't even notice the peeking lighthouse I just thought "huh that's weird they don't give it a name" 😆

**Comment by patrickhulce** [comment]
anyone else want to be a part of the approval magic? @connorjclark @paulirish  you did a lot of the lifting here care to do the honors :)

**Pull request merged**